consensus/clique/clique.go

```diff
@@ -137,8 +137,9 @@ var (
 	errRecentlySigned = errors.New("recently signed")
 )
 
-// SignerFn is a signer callback function to request a header to be signed by a
-// backing account.
+// SignerFn hashes and signs the data to be signed by a backing account. It should point to the
+// accounts.Wallet.SignData method.
```

Comment:
This is an interface method, it should not point to anything explicit. It's up to the caller. Please delete that part.

Reply:
It's highly coupled with `Wallet.SignData` behaviour, notably hashing the message before signing. The current `Clique` implementation depends on it:

+ `SignData` will hash the data under the hood (without your knowledge)
+ `ecrecovery` will not hash, and if you forget about it you will have a wrong result.

Reply:
This is an interface, whether the current mplementation is a specific code is not relevant.

Reply:
I've updated the comment. I added a note about current implementation, but if you want I can remove it.